source/userguide/auto_annotator.rst

```diff
@@ -0,0 +1,489 @@
+			'grain': 'day',
+			'type': 'value'},
+			'dim': 'sys_time'
+		}
+	]
+
+The Auto Annotator detected "Apple" as :attr:`sys_org`. Moreover, it recognized "$10" as :attr:`sys_amount-of-money` and resolved its :attr:`value` as 10 and :attr:`unit` as "$".
+Lastly, it recognized "last monday" as :attr:`sys_time` and resolved its :attr:`value` to be a timestamp representing the last monday from the current date.
+
+In general, detected entities will be represented in the following format:
+
+.. code-block:: python
+
+	entity = {
+
+		"body": (substring of sentence), 
+		"start": (start index), 
+		"end": (end index + 1), 
+		"dim": (entity type), 
+		"value": (resolved value, if it exists), 
+
+	}
+
+To restrict the types of entities returned from the :attr:`parse()` method use the :attr:`entity_types` parameter and pass in a list of entities to restrict parsing to. By default, all entities are allowed.
+For example, we can restrict the output of the previous example by doing the following:
+
+
+.. code-block:: python
+	
+	allowed_entites = ["sys_org", "sys_amount-of-money", "sys_time"]
+	sentence = "Apple stock went up $10 last monday."
+	sa.parse(sentence=sentence, entity_types=allowed_entities) 
+
+Auto Annotator Configuration
+----------------------------
+
+The :attr:`DEFAULT_AUTO_ANNOTATOR_CONFIG` shown below is the default config for an Annotator.
+A custom config can be included in :attr:`config.py` by duplicating the default config and renaming it to :attr:`AUTO_ANNOTATOR_CONFIG`.
+Alternatively, a custom config dictionary can be passed in directly to :class:`SpacyAnnotator` or any Annotator class upon instantiation.
+
+
+.. code-block:: python
+
+	DEFAULT_AUTO_ANNOTATOR_CONFIG = { 
+
+		"annotator_class": "SpacyAnnotator",
+		"overwrite": False, 
+		"annotate": [ 
+			{ 
+				"domains": ".*", 
+				"intents": ".*", 
+				"files": ".*", 
+				"entities": ".*", 
+			} 
+		], 
+		"unannotate_supported_entities_only": True, 
+		"unannotate": None, 
+	}
+
+Let's take a look at the allowed values for each setting in an Auto Annotator configuration.
+
+
+``'annotator_class'`` (:class:`str`): The class in auto_annotator.py to use for annotation when invoked from the command line. By default, :class:`SpacyAnnotator` is used. 
+
+``'overwrite'`` (:class:`bool`): Whether new annotations should overwrite existing annotations in the case of a span conflict. False by default. 
+
+``'annotate'`` (:class:`list`): A list of annotation rules where each rule is represented as a dictionary. Each rule must have four keys: :attr:`domains`, :attr:`intents`, :attr:`files`, and :attr:`entities`.
+Annotation rules are combined internally to create Regex patterns to match selected files. The character :attr:`.*` can be used if all possibilities in a section are to be selected, while possibilities within
+a section are expressed with the usual Regex special characters, such as :attr:` . ` for any single character and :attr:`|` to represent "or". 
```

Comment:
I think the `.` character is not rendered correctly
![image](https://user-images.githubusercontent.com/1332283/95546287-0571b280-09b5-11eb-9735-52e1c064c849.png)